spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py

```diff
@@ -311,6 +311,9 @@ def __init__(self, stackdriver, spectator_response_processor):
         'summarize_compound_kinds')
     self.__distributions_also_have_count = stackdriver.stackdriver_options.get(
         'distributions_also_have_count')
+    # When you are familiar with the internal metrics, using the display name
+    # get's confusing so let's require that to be explicitly enabled.
+    self.__add_display_name = stackdriver.stackdriver_options.get('add_display_name')
```

Comment:
Does this need to be added as a flag or does this flag already exist?

Reply:
It's not intended as a flag, just in spinnaker-monitoring.yml.

The "stackdriver_options" is actually nested in "stackdriver", not the root level.